src/Modules/Backend/Domain/User/Command/CreateUserHandler.php

```diff
@@ -5,6 +5,7 @@
 use ForkCMS\Core\Domain\MessageHandler\CommandHandlerInterface;
 use ForkCMS\Modules\Backend\Domain\User\User;
 use ForkCMS\Modules\Backend\Domain\User\UserRepository;
+use Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Google\GoogleAuthenticatorInterface;
```

Comment:
Is this use statement actually used?

Reply:
fixed 26235e64cfdc8b934a229fb49c8bee19a6cf6f2b